### PR TITLE
basic nav and about me page

### PR DESCRIPTION
**Pull request by scrimmie**
Adding a base navigation to the application.

This also adds a preliminary about me page.

Note: This has not been made responsive yet

### CODE REVIEW

**Comment by ethanreyna** [comment]
cringe and weird

**Comment by Jacobsmi** [comment]
SVGs should be moved to their own files to clean up markup code.

**Comment by Jacobsmi** [comment]
> SVGs should be moved to their own files to clean up markup code.

Also makes them more reusable

File: src/components/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  TfiUser,
+  TfiFolder,
+  TfiCamera,
+  TfiBrushAlt,
+  TfiEmail,
+} from "react-icons/tfi";
+import { IoCode } from "react-icons/io5";
+
+export const Navigation = () => {
+  const navLinks = [
```
Comment (Jacobsmi) [inline src/components/Navigation.tsx:12]: Type?

File: src/components/Navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          </div>
+          {name}
+        </a>
+      ))}
+    </div>
+  );
+
+  return (
+    <nav className="w-full flex justify-center">
+      <NavLinks className=" flex flex-row gap-10 text-md" />
```
Comment (Riccardo-Maio) [inline src/components/Navigation.tsx]:
```suggestion
      <NavLinks className="flex flex-row gap-10 text-md" />
```

**Review by Jacobsmi — CHANGES_REQUESTED** [review]

File: src/components/Navigation.tsx (hunk 1 above)
Comment (scrimmie) [reply src/components/Navigation.tsx:12]: This is auto inferred.

**Pull request merged**